Singularity/Singularity/Manager/DistributionManager.cs

```diff
@@ -68,7 +72,7 @@ public DistributionManager()
             mRequestedUnitsProduce = new Queue<Task>();
             mRequestedUnitsDefense = new Queue<Task>();
 
-            //Other stuff
+            //Other stuff -- ???
```

Comment:
Nein dieser kommentar war nötig ;D

Reply:
Is ja immernoch da, nur mit Fragezeichen versehen :D :P